.github/workflows/publish-prod.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         with:
           switches: -avzr --delete
           path: build/
-          remote_path: ${{ secrets.AWS_PROD_PUBLISH_PATH }}
+          remote_path: ${{ secrets.AWS_PROD_PUBLISH_PATH_UNSUPPORTED }}/0.25
```

Comment:
I've already added this new key to the project secrets (and to the Keeper record that contains these secrets).